Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py

```diff
@@ -217,10 +263,9 @@ def test_module(client: Client, *_) -> Tuple[str, Dict[Any, Any], List[Any]]:
             if limit < 1 or limit > 50:
                 msg += 'Fetch Limit value should be between 1 and 50. '
 
-        if params.get('fetch_parameters') == 'ID and timestamp':
-            if not (params.get('column_name') and params.get('id_column')):
-                msg += 'Missing Fetch Column or ID Column name (when ID and timestamp are chosen,' \
-                       ' fill in both). '
+        if params.get('fetch_parameters') == 'ID and timestamp' and not (params.get('column_name') and params.get('id_column')):
```

Comment:
please add brackets where applicable to avoid confusion `(if x and (not y) and z)` vs `(if x and not (y and z)`